Sources/Parley/Parley.swift

```diff
@@ -788,7 +795,7 @@ extension Parley {
        - localizationManager: Manager to return localization string from a key.
      */
     public static func setLocalizationManager(_ localizationManager: LocalizationManager) {
-        shared.localizationManager = localizationManager
+        ParleyLocalizationKey.localizationManager = localizationManager
```

Comment:
During running unit tests I had an issue that the localzationManager was set to a spy and not being reset. That gave me the idea to keep the localizationManager in the `ParleyLocalizationKey`.